shimmer/develop/sidebars.ts

```diff
@@ -1,200 +1,216 @@
 module.exports = {
   develop: [
-    'welcome',
     {
-      type: 'link',
-      label: 'Protocol Overview',
-      href: '/introduction/welcome',
-    },
-    {
-      type: 'link',
-      label: 'Tangle Improvement Proposals (TIPS)',
-      href: '/tips',
+      type: 'category',
+      label: 'What is Shimmer?',
+      link: {
+        type: 'doc',
+        id: 'what-is-shimmer/introduction',
+      },
+      items: [
+        'what-is-shimmer/layer-1-landscape',
+        'what-is-shimmer/networks',
+        {
+          type: 'category',
+          label: 'About Nodes',
+          collapsed: true,
+          link: {
+            type: 'doc',
+            id: 'what-is-shimmer/nodes/about-nodes',
+          },
+          items: [
+            'what-is-shimmer/nodes/hornet',
+            'what-is-shimmer/nodes/become-a-node-operator',
+          ],
+        },
+        'what-is-shimmer/clients',
+        'what-is-shimmer/tokens-and-wallets',
+      ],
     },
+
     {
       type: 'category',
-      label: 'Smart Contracts',
+      label: 'Node API Reference',
       collapsed: true,
       items: [
+        'nodes/api_reference',
         {
-          label: 'Documentation',
-          type: 'link',
-          href: '/smart-contracts/overview',
-        },
-        {
-          label: 'Node (WASP)',
-          type: 'link',
-          href: '/smart-contracts/guide/chains_and_nodes/running-a-node',
-        },
-        {
-          type: 'link',
-          label: 'WASP CLI',
-          href: '/smart-contracts/guide/chains_and_nodes/wasp-cli',
-        },
-        {
-          label: 'Schema Tool',
-          type: 'link',
-          href: '/smart-contracts/guide/wasm_vm/schema',
+          type: 'category',
+          label: 'Core REST API',
+          collapsed: true,
+          link: {
+            type: 'doc',
+            id: 'nodes/core-rest-api/iota-core-rest-api',
+          },
+          items: require('./nodes/core-rest-api/sidebar'),
         },
       ],
     },
+
     {
       type: 'category',
-      label: 'Libraries',
+      label: 'Public Node Endpoints',
```

Comment:
It's now called just "Endpoints" in the current wiki. But I don't mind changing it

Reply:
Endpoints works just as well and it's shorter :)